apps/web/src/views/Farms/components/FarmCard/CardHeading.tsx

```diff
@@ -79,6 +81,7 @@ const CardHeading: React.FC<React.PropsWithChildren<ExpandableSectionProps>> = (
   const { t } = useTranslation()
   const chainId = useChainId()
   const isReady = multiplier !== undefined || bCakeWrapperAddress
+  const hasSwellReward = useHasSwellReward(lpAddress ?? '0x0', 'FARM')
```

Comment:
if we allow nullable value for the hook, we don't need to add default value here
```suggestion
  const hasSwellReward = useHasSwellReward(lpAddress)
```